service/controller/v19/key/key.go

```diff
@@ -438,6 +439,17 @@ func SmallCloudConfigS3URL(customObject v1alpha1.AWSConfig, accountID string, ro
 	return fmt.Sprintf("s3://%s", SmallCloudConfigPath(customObject, accountID, role))
 }
 
+// SubnetID generates Crockford Base32 (http://www.crockford.com/wrmg/base32.html)
+// encoded representation of subnet CIDR to be used as a shared identifier for
+// specific network e.g. in CloudFormation template resource names.
+func SubnetID(subnet *net.IPNet) string {
+	id := subnet.String()
+	id = strings.Replace(id, ".", "o", -1)
+	id = strings.Replace(id, "/", "m", -1)
+	id = strings.Replace(id, ":", "x", -1)
```

Comment:
Is there any reasoning behind choosing those letter replacements or this is all random?